recipes/jasper/all/conanfile.py

```diff
@@ -39,12 +41,20 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
         del self.settings.compiler.cppstd
         del self.settings.compiler.libcxx
 
+        # Handle deprecated libjpeg option
+        self.output.warn("jpegturbo option is deprecated, use with_libjpeg option instead.")
+        if self.options.with_libjpeg == "libjpeg" and self.options.jpegturbo:
+            self.options.with_libjpeg == "libjpeg-turbo"
```

Comment:
Copy Paste bugs :speak_no_evil: 
```suggestion
            self.options.with_libjpeg = "libjpeg-turbo"
```

Reply:
good catch